plugin/trino-iceberg/src/main/java/io/trino/plugin/iceberg/IcebergMetadata.java

```diff
@@ -1137,6 +1137,7 @@ public Optional<ConnectorOutputMetadata> finishInsert(ConnectorSession session,
                     .withFileSizeInBytes(task.fileSizeInBytes())
                     .withFormat(table.getFileFormat().toIceberg())
                     .withMetrics(task.metrics().metrics());
+            task.fileSplitOffsets().ifPresent(builder::withSplitOffsets);
```

Comment:
how does this builder relates to `IcebergPageSink`?

Reply:
Builder is an existing iceberg API, we're just using it to populate the offsets that we received from workers through `CommitTaskData`